src/components/Homepage.js

```diff
@@ -229,35 +234,19 @@ function Homepage() {
               margin: '24px',
             }}
           >
-            <svg
-              version="1.1"
-              id="Layer_1"
-              xmlns="http://www.w3.org/2000/svg"
-              xmlnsXlink="http://www.w3.org/1999/xlink"
-              x="0px"
-              y="0px"
-              viewBox="0 0 64 64"
-              style={{ enableBackground: 'new 0 0 64 64' }}
-              xmlSpace="preserve"
+            <a href="https://github.com/codeclannigeria" target="_blank">
```

Comment:
Codacy found an issue: [Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=6048149)